src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/CommandBarTests/UnoSamples_Tests.NativeCommandBar.cs

```diff
@@ -203,5 +203,36 @@ public void When_Navigated_CommandBarShouldKeepColor_Native_Frame()
 
 			ImageAssert.AreEqual(initial, final);
 		}
+
+		[Test]
+		[AutoRetry]
+		[ActivePlatforms(Platform.iOS)]
+		public void When_Navigated_CommandBarShouldHideBackButtonTitle_NativeFrame()
+		{
+			Run("UITests.Windows_UI_Xaml_Controls.CommandBar.BackButtonTitle.CommandBar_Frame");
+
+			_app.WaitForElement("NavigateInitialButton");
+
+			// Will set the global style for the CommandBar to remove the Back Button Title
+			_app.FastTap("SetGlobalStyleButton");
+
+			_app.FastTap("NavigateInitialButton");
+
+			_app.WaitForElement("NavigateToPage2Button");
+			_app.FastTap("NavigateToPage2Button");
+
+			_app.WaitForElement("BackButtonTitleButton");
+
+			_app.FastTap("BackButtonTitleButton");
+
+			_app.Wait(TimeSpan.FromMilliseconds(800));
```

Comment:
You should be able to use `App.WaitForDependencyPropertyValue()` to avoid arbitrarily waiting some time (it may fail if the underlying machine is slow).

Reply:
`App.WaitForDependencyPropertyValue()`This one is new for me.. Will update. Thanks :)

Reply:
Updated!